tests/util.py

```diff
@@ -0,0 +1,21 @@
+import copy
+
+
+def strip_uid_from_dict(node_dict):
+    """
+    Remove "uid" attributes from nested dictionaries.
```

Comment:
I need more explanation to understand what you're testing here

Reply:
What are you missing.
The doc says, that I am removing all uid attributes.

uids are easily different, so for test purposes sometimes we want to make sure that we can ignore this.
Hence this helper funciton.